scripts/qa_crowbarsetup.sh

```diff
@@ -1233,12 +1241,23 @@ function create_repos_yml()
         SUSE-Enterprise-Storage-2-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/Storage:/2:/x86_64/update/ \
         >> $tmp_yml
```

Comment:
This bit can probably be removed; SLES 12 GA is not even supported for Cloud 6 anymore either.